packages/library-legacy/test/connection.test.ts

```diff
@@ -5175,14 +5147,14 @@ describe('Connection', function () {
             mint: testTokenMintPubkey,
           })
         ).value;
-        expect(accountsWithMintFilter).to.have.length(2);
+        expect(accountsWithMintFilter).to.have.length(1);
 
         const accountsWithProgramFilter = (
           await connection.getTokenAccountsByOwner(testOwnerKeypair.publicKey, {
             programId: TOKEN_PROGRAM_ID,
           })
         ).value;
-        expect(accountsWithProgramFilter).to.have.length(3);
+        expect(accountsWithProgramFilter).to.have.length(1);
```

Comment:
This is not _exactly_ the same as the prior test, but it's close enough. We're not meant to be testing the underlying RPC here.